integration-msk-to-lambda/example.php

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+<?php
+
+# using bref/bref and bref/logger for simplicity
+
+use Bref\Context\Context;
+use Bref\Event\Kafka\KafkaEvent;
+use Bref\Event\Handler as StdHandler;
+use Bref\Logger\StderrLogger;
+
+require __DIR__ . '/vendor/autoload.php';
+
+class Handler implements StdHandler
+{
+    private StderrLogger $logger;
+    public function __construct(StderrLogger $logger)
+    {
+        $this->logger = $logger;
+    }
+
+    /**
+     * @throws JsonException
+     * @throws \Bref\Event\InvalidLambdaEvent
+     */
+    public function handle(mixed $event, Context $context): void
+    {
+        $kafkaEvent = new KafkaEvent($event);
+        $this->logger->info("Processing records");
```

Comment:
Can you please match the print statements to the other snippets in this suite: i.e. https://github.com/aws-samples/serverless-snippets/blob/main/integration-msk-to-lambda/Example.java Thank you!

Reply:
Added the logic that matches Java version.

Since bref currently doesn't support `getKey` getter, please do not merge until this is resolved: https://github.com/brefphp/bref/pull/1862

Reply:
Merged now